### PR TITLE
Travis problem fixes

### PR DESCRIPTION
**Pull request by otaviobp**

### CODE REVIEW

**Comment by karthikprabhuvinod** [comment]
Any reason why you switched to bionic?

File: .travis.yml [hunk 1]
```diff
@@ -46,30 +46,37 @@ jobs:
 # - the Swupd_Root.pem cert must be installed out-of-tree to run the test suite with signature verification enabled
 # - python3-docutils for the rst2man script
 install:
-        - wget https://github.com/libcheck/check/releases/download/0.11.0/check-0.11.0.tar.gz
-        - tar -xvf check-0.11.0.tar.gz
-        - pushd check-0.11.0 && ./configure --prefix=/usr && make -j48 && sudo make install && popd
+        # Install dependencies from ubuntu
+        - sudo apt-get install python3-docutils
+        - sudo apt-get install shellcheck
```
Comment (karthikprabhuvinod) [inline .travis.yml]: Maybe we can combine then by using commas

File: test/code_analysis/compliant.bats [hunk 2]
```diff
@@ -26,6 +26,7 @@ check_sort_makefile()
 
 	run clang-format-9 -i -style=file src/*.[ch] src/lib/*.[ch]
 	if [ "$status" -ne 0 ]; then
+		echo "clang-format-9 failed with status $status. Check if you have clang-format-9 installed"
```
Comment (castulo) [inline test/code_analysis/compliant.bats]: 🤣 learn from my mistakes

**Comment by otaviobp** [comment]
> Any reason why you switched to bionic?

I added that to the commit message.
 - clang-format was broken for xenial
 - we can use libcheck from ubuntu instead of building from source

And it's better to run that in a more updated system.

**Pull request merged**